examples/PromptInjectionDetector/PromptInjectionGPT4o.ipynb

```diff
@@ -0,0 +1,439 @@
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "c0hvsPRZLCUz"
+   },
+   "source": [
+    "First, we'll install some dependencies.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "rTIA4DVQw1K7"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": []
+    }
+   ],
+   "source": [
+    "%pip install -U braintrust openai datasets autoevals python-dotenv --quiet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll load our environmental variables which has OpenAI and Brain Trust keys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "load_dotenv() "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll import the [prompt-injections dataset](https://huggingface.co/datasets/deepset/prompt-injections) from Huggingface.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import braintrust\n",
+    "import os\n",
+    "\n",
+    "from openai import OpenAI\n",
+    "\n",
+    "client = braintrust.wrap_openai(\n",
+    "    OpenAI(api_key=os.environ.get(\"OPENAI_API_KEY\", \"<Your OpenAI Key Here>\"))\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "id": "8UDe2_sAw1K7"
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Using the latest cached version of the dataset since deepset/prompt-injections couldn't be found on the Hugging Face Hub\n",
+      "Found the latest cached dataset configuration 'default' at /Users/nelsonauner/.cache/huggingface/datasets/deepset___prompt-injections/default/0.0.0/3e7f39ebf36efb8e69bf236623138cb7567acb60 (last modified on Mon May 20 12:53:54 2024).\n"
+     ]
+    }
+   ],
+   "source": [
+    "from datasets import load_dataset\n",
+    "\n",
+    "# Load dataset from Huggingface.\n",
+    "dataset = load_dataset(\"deepset/prompt-injections\", split=\"train\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "N_DATAPOINTS = 30\n",
+    "SEED=17\n",
+    "\n",
+    "\n",
+    "# Shuffle and trim to 20 datapoints. Restructure our dataset\n",
+    "# slightly so that each item in the list contains an input\n",
+    "# being the title and the expected category index label.\n",
```

Comment:
"contains an input being the title" sounds a bit off. Maybe "each item in the list contains a title as input and category as expected"